app/src/main/res/values-fr/strings.xml

```diff
@@ -609,23 +609,23 @@
     <string name="yesterday">Hier</string>
     <string name="some_days_ago">Il y a %d jours</string>
     <plurals name="second">
-        <item quantity="one">sec</item>
+        <item quantity="one">%d sec</item>
```

Comment:
I can't read French. Does this change break grammar, or make the whole text weird?

Reply:
tbh I am not sure. But Android's resources on plurals mentioned that one should always supply the argument %d in plurals.

> Always include %d in "one" because translators will need to use %d for languages where "one" doesn't mean 1 (as explained above).

Taken from https://developer.android.com/guide/topics/resources/string-resource#Plurals